### ISSUE TITLE
Hide System Tray (and/or other options)

### ISSUE THREAD

**Issue by lbell** [QTodoTxt/QTodoTxt#80]
Would be great to be able to hide the system tray. At least on Ubuntu 14.04 it is big, out of place, and doesn't do anything.

**Comment by tristan-k** [comment]
+1 

There is a comment stating that it is just a matter of fixing one line: http://www.webupd8.org/2014/10/manage-your-tasks-with-qtodotxt-cross.html#comment-1623716441

There is also another indicator which works properly with Ubuntu 14.04: http://www.webupd8.org/2014/04/todo-indicator-ubuntu-appindicator-for.html

**Comment by orschiro** [comment]
+1 from me to disable the tray icon since it doesn't provide any functionality on Ubuntu 15.04 either. Can we remove the icon by commenting out the following lines in `app.py`:

```
from qtodotxt.ui.views.main_view import MainView
14	
15	
16	class TrayIcon(QtGui.QSystemTrayIcon):
17	    def __init__(self, main_controller):
…	
24	        QtGui.QSystemTrayIcon.__init__(self, icon, view)
25	        self.activated.connect(self._onActivated)
26	        self.setToolTip('QTodoTxt')
```

Search query: https://github.com/mNantern/QTodoTxt/search?utf8=%E2%9C%93&q=tray

**Comment by thomas-mc-work** [comment]
This icon would be useful if
- I could minimize (optionally via the close button) the application only leaving the tray icon
- on click on the tray icon the window will be restored

**Comment by oroulet** [comment]
new settings merged in 1.6beta. try it with pip install QTodoTxt --pre
